### ISSUE TITLE
missing assertions for global dependencies

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#210]
Related to code review #178.

Tasks/functions that rely on the existence of globals should assert that they exist. 

The following global dependences were recently introduced into afterRequirejsBuild.js without adding corresponding assertions:

global.imageAndAudioLicenseInfo
global.phet.chipper
global.phet.chipper.getBrand

Assertions should be added to the set that appears after the comment at line 44:

`// globals that should be defined by this point`

**Comment by pixelzoom** [comment]
Note that this should be done after renaming `global.imageAndAudioLicenseInfo`, see https://github.com/phetsims/chipper/issues/211

**Comment by samreid** [comment]
Fixed above.  When we rename imageAndAudioLicenseInfo in #212 we can rename this as well.  I skimmed through the rest of chipper js as well and many of the other occurrences of `global` look like assignments.  One exception is mipmap.js which has:

```js
        global.phet.mipmapsToBuild.push( {
          name: name,
          path: path,
          level: options.level,
          quality: options.quality
        } )
```
where mipmapsToBuild is created in beforeRequirejsBuild.js.  However, in this case, the mipmap.js would immediately fail at the push call, I'm not sure whether an assertion a line above that would be helpful enough.

@pixelzoom, back to you.
